broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -249,45 +343,229 @@ class ServiceFabrikApiController extends FabrikBaseController {
           .invoke()
           .tap(response => logger.info('backup response ', response))
           .then(body => res
-            .status(202)
+            .status(CONST.HTTP_STATUS_CODE.ACCEPTED)
             .send(body)
           );
       });
   }
 
+  getBackupOptions(backup_guid, deployment, req) {
+    return Promise
+      .all([
+        cf.cloudController.findServicePlanByInstanceId(req.params.instance_id),
+        cf.cloudController.getOrgAndSpaceGuid(req.params.instance_id)
+      ])
+      .spread((plan_details, res) => {
```

Comment:
Don't use snake case for `plan_details`. rename `res` to `orgAndSpaceDetails`

Reply:
Done